src/components/pages/DonationConfirmation.vue

```diff
@@ -1,47 +1,56 @@
 <template>
 	<div class="donation-confirmation">
 		<a class="mobile-call-to-action is-primary button" href="#membership-application-url"
-			v-on:click="scrollToCallToAction"
-			v-if="isMobileCallToActionButtonVisible && !isAddressModalOpen">
+			v-if="isMobileCallToActionButtonVisible && !isAddressModalOpen"
+			v-on:click="scrollToCallToAction">
 			Jetzt Fördermitglied werden
-			<chevron-down-icon/>
+			<ChevronDownIcon/>
 		</a>
 
 		<div class="columns is-multiline is-variable is-2">
 			<div class="column is-full pt-0 pb-0">
-				<success-message-bank-transfer v-if="showBankTransferContent" :donation="donation"/>
-				<success-message
+				<SuccessMessageBankTransfer v-if="showBankTransferContent" :donation="donation"/>
+				<SuccessMessage
 					v-else
 					:donation="donation"
 					:comment-link-is-disabled="commentLinkIsDisabled"
 					v-on:show-comment-modal="showCommentModal()"
 				/>
 			</div>
-			<div class="column is-half pt-0 pb-0" v-if="!donation.isExported">
-				<address-known
-					v-if="showAddress"
-					v-on:show-address-modal="showAddressModal()"
-					:donation="donation"
-					:address="currentAddress"
+			<!-- Left Column -->
```

Comment:
I'm guessing these HTML comments were added when you were trying to understand the placement of the confirmation elements? You should remove them now though.